### PR TITLE
Add common labels handling

### PR DESCRIPTION
**Pull request by sebastienmusso**

### CODE REVIEW

**Comment by yonatankahana** [comment]
hi @sebastienmusso, 
can you please bump the helm chart version? it will trigger auto release process once merged

File: charts/k8tz/templates/_helpers.tpl [hunk 1]
```diff
@@ -56,6 +56,9 @@ helm.sh/chart: {{ include "k8tz.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- if .Values.commonLabels}}
```
Comment (yonatankahana) [inline charts/k8tz/templates/_helpers.tpl:59]: please add space before the `}}`

**Review by yonatankahana — CHANGES_REQUESTED** [review]

File: charts/k8tz/values.yaml [hunk 2]
```diff
@@ -7,6 +7,9 @@ injectionStrategy: initContainer
 timezone: UTC
 injectAll: true
 
+# Labels to apply to all resources
+commonLabels: {}
```
Comment (yonatankahana) [inline charts/k8tz/values.yaml:11]: helm chart values should also be documented in the readme

**Review by yonatankahana — CHANGES_REQUESTED** [review]

**Review by yonatankahana — COMMENTED** [review]
what about the labels of pods created by the Deployment?
the template labels are `k8tz.selectorLabels` (since using `k8tz.labels` will force redundant re-deployments every chart version change).

https://github.com/k8tz/k8tz/blob/50729bd0e68659a1df91fcf390948bc243d9c7b5/charts/k8tz/templates/deployment.yaml#L20-L21

**Comment by sebastienmusso** [comment]
> what about the labels of pods created by the Deployment?
> the template labels are `k8tz.selectorLabels` (since using `k8tz.labels` will force redundant re-deployments every chart version change).
> 
> https://github.com/k8tz/k8tz/blob/50729bd0e68659a1df91fcf390948bc243d9c7b5/charts/k8tz/templates/deployment.yaml#L20-L21

Do I have to add commonLabels to selectorLabels too ?

**Comment by yonatankahana** [comment]
> Do I have to add commonLabels to selectorLabels too ?

no, only the selector labels should use for matching

**Pull request closed without merging**